apps/resources/cs.json

```diff
@@ -10,7 +10,7 @@
   "name": "cs",
   "prebuiltBinaries": {
     "x86_64-pc-linux": "gz+https://github.com/coursier/coursier/releases/download/v${version}/cs-x86_64-pc-linux.gz",
-    "aarch64-pc-linux": "gz+https://github.com/coursier/coursier/releases/download/v${version}/cs-aarch64-pc-linux.gz",
+    "aarch64-pc-linux": "gz+https://github.com/VirtusLab/coursier-m1/releases/download/v${version}/cs-aarch64-pc-linux.gz",
```

Comment:
That only applies to `2.1.0-RC5` and later versions. Maybe we should put the former URL in a [version override](https://github.com/coursier/apps/blob/fd91e1a23fea4abcdb4a1c490c5b0dd1e03f046d/apps/resources/scala.json#L12).

Reply:
Yes, that’s a good point